secrets-provider/templates/role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: {{ include "conjur.rbac-api" . }}
```

Comment:
Missing `{{- if .Values.rbac.create -}}`

Reply:
😲 you are right!